packages/core/src/form-field/FormFieldControlWrapper.css

```diff
@@ -1,7 +1,18 @@
 .saltFormFieldControlWrapper {
   align-items: center;
+  border: none;
   display: grid;
   grid-auto-columns: minmax(min-content, 1fr);
   grid-auto-flow: column;
   grid-column-gap: var(--salt-size-unit);
+  margin: 0;
+  padding: 0;
+}
+
+.saltFormFieldControlWrapper-secondary .saltInput {
```

Comment:
would it be better to maybe pass the variant to the input?

Reply:
yes, updated